static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -6,36 +8,39 @@
 }
 
 .new-header__logo {
+    padding-top: ($gs-baseline / 2);
+    padding-bottom: ($gs-baseline / 2);
 
     svg {
```

Comment:
Did we not add the functionality to add a class to an `svg`? or is the svg added via JS which is why you still target the element directly?